processor/src/main/java/org/mapstruct/ap/internal/model/NestedTargetPropertyMappingHolder.java

```diff
@@ -229,6 +233,13 @@ public NestedTargetPropertyMappingHolder build() {
                         handledTargets.add( entryByTP.getKey().getName() );
                     }
 
+                    handleSourceParameterMappings(
```

Comment:
Yes, with this one for some reason a source parameter is into the groupings, but it was not included in the mappings before I assume that the source parameter is mapped to the target property. For this reason the mappings are empty (if they are not it would be treated as properties of the parameter)